crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -253,6 +254,43 @@ pub enum Condition {
     SkipRelativeCondition,
 }
 
+fn check_agg_sig_unsafe_message(
+    a: &Allocator,
+    msg: NodePtr,
+    constants: &ConsensusConstants,
+) -> Result<(), ValidationErr> {
+    if a.atom_len(msg) < 32 {
+        return Ok(());
+    }
+    let buf = a.atom(msg);
+    if buf
```

Comment:
I'd prefer this iterate over the constants rather than be repeated like this, if possible.

Reply:
I suppose it would be unrolled anyway